gensim/models/experimental/drmm_tks.py

```diff
@@ -0,0 +1,595 @@
+import logging
+import random
+import numpy
+
+import numpy as np
+from gensim.scripts.glove2word2vec import glove2word2vec
+from gensim.test.utils import get_tmpfile
+from gensim.models import KeyedVectors
+from collections import Counter
+from custom_losses import rank_hinge_loss
+from custom_layers import TopKLayer
+from sklearn.preprocessing import normalize
+from custom_callbacks import ValidationCallback
+from gensim import utils
+
```

Comment:
No module docstring (what is it, why this needed, examples of usage), have a look to a good example 
https://github.com/RaRe-Technologies/gensim/blob/fd6f6eb71996cd36a6236070f33efdfb37dde342/gensim/models/keyedvectors.py#L8-L147